client/common/client.c

```diff
@@ -1673,6 +1673,11 @@ static BOOL freerdp_handle_touch_up(rdpClientContext* cctx, const FreeRDP_TouchC
 			const UINT32 contactFlags = ((contact->flags & FREERDP_TOUCH_HAS_PRESSURE) != 0)
 			                                ? CONTACT_DATA_PRESSURE_PRESENT
 			                                : 0;
+			// Ensure contact position is unchanged from "engaged" to "out of range" state
+			rdpei->TouchRawEvent(rdpei, contact->id, contact->x, contact->y, &contactId,
```

Comment:
sure you want to call the event 2 times instead of just adding the flags?

Reply:
This is similar to the `rdpei_touch_end` implementation.
In my environment it also works fine without it.
This is for compliance with the MS-RDPEI specification.